ansi/writer.go

```diff
@@ -1,46 +1,46 @@
 package ansi
 
 import (
+	"bytes"
 	"io"
-	"strings"
 )
 
 type Writer struct {
 	Forward io.Writer
 
 	ansi       bool
-	ansiseq    string
-	lastseq    string
+	ansiseq    bytes.Buffer
+	lastseq    bytes.Buffer
 	seqchanged bool
 }
 
 // Write is used to write content to the ANSI buffer.
 func (w *Writer) Write(b []byte) (int, error) {
-	for _, c := range string(b) {
```

Comment:
The change to iterating over bytes instead of runes seems to break glamor's UTF-8 processing here: https://github.com/charmbracelet/glamour/blob/master/ansi/margin.go

The first indent pipe (using ansi.Writer) breaks up UTF-8 byte sequences into single byte writes, which are misinterpreted when ingested into the second padding pipe, as they go through a single byte->string->[]byte conversion.

Here's an illustration:
https://play.golang.org/p/N-caKzuAwdQ

Reply:
Hey @zavislak I don't know why you use `bad = append(bad, []byte(string(c))[0])`

You can see this demo instead. https://play.golang.org/p/fHX2TW2-ORG

Reply:
I believe you can build glow at head and see the broken UTF-8 sequences, which led me to try track this down to happening between the indent and padding pipes in glamor.  If glow properly handles utf-8 for you at head, please let me know.